### DISCUSSION TITLE
Grep on Mobile

### DISCUSSION THREAD

**Discussion by jin-ahn** [tadashi-aikawa/obsidian-another-quick-switcher#193] [Q&A]
What are the options to get grep working on mobile? I want to be able to search for a specific word across my vault, but I don't want to have to install or use a separate search plugin to do just that.

**Comment by tadashi-aikawa** [comment]
Sorry for my late answer, @jin-ahn :)

Please see https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/196.
